.github/CODEOWNERS

```
@@ -2,15 +2,19 @@
 ##### Global Protection Rule ######
 ###################################
 # NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below.
-*                                               @hashgraph/devops-ci @hashgraph/release-engineering-managers
+*                                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers
 
 #########################
 ####   Java Modules  ####
 #########################
-/hedera-cryptography-altbn128/                  @hashgraph/platform-hashgraph @hashgraph/platform-base
-/hedera-cryptography-pairings-api/              @hashgraph/platform-hashgraph @hashgraph/platform-base
-/hedera-cryptography-pairings-signatures/       @hashgraph/platform-hashgraph @hashgraph/platform-base
-/hedera-common-nativesupport/                   @hashgraph/platform-hashgraph @hashgraph/platform-base
+/cryptography/hedera-cryptography-altbn128/             @hashgraph/platform-hashgraph @hashgraph/platform-data @hashgraph/platform-base @hashgraph/platform-architects
+/cryptography/hedera-cryptography-pairings-api/         @hashgraph/platform-hashgraph @hashgraph/platform-data @hashgraph/platform-base @hashgraph/platform-architects
+/cryptography/hedera-cryptography-pairings-signatures/  @hashgraph/platform-hashgraph @hashgraph/platform-data @hashgraph/platform-base @hashgraph/platform-architects
```

Since these are identical it can be simplified to a single rule.

```suggestion
/cryptography/         @hashgraph/platform-hashgraph @hashgraph/platform-data @hashgraph/platform-base @hashgraph/platform-architects
```

---

thanks! I wasn't sure if it was going to work!